spec/features/home_page_spec.rb

```diff
@@ -1,8 +1,33 @@
 require 'rails_helper'
 
-describe 'Home Page' do
+feature 'Home Page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.